src/components/tooltip/index.css

```diff
@@ -0,0 +1,9 @@
+.hx-tooltip-label  {
+    border-bottom: 1px dotted red;
```

Comment:
I forgot to go over theming with you. We need to move everything that isn't part of the 'theme' (e.g. `display`, `overflow` stuff like that) into this file and add variables for everything else.

In this case in `index.css`
```css
.hx-tooltip-label {
  border-bottom: 1px dotted transparent;
}
```

Then inside `theme.css`
```css
.hx-tooltip-label {
  border-bottom-color: var(--tooltip-border-color);
}
```

Variable definitions then go into `hexagon.variables.css`. The file is in alphabetical order and there is a heading for each component/module

Something like this might suit:
```css
/* Tooltip */
--tooltip-border-color: var(--theme-accent-color);
```